spec/views/devise/sessions/new.html.erb_spec.rb

```diff
@@ -244,6 +245,26 @@
         expect(rendered).to have_button(t('links.sign_in'))
         expect(rendered).to have_css('lg-captcha-submit-button')
       end
+
+      before do
+        allow(FeatureManagement).to receive(:sign_in_recaptcha_enabled?).and_return(true)
+      end
+
```

Comment:
This should already be stubbed above?

https://github.com/18F/identity-idp/blob/d37dbc8326c1694d0a7da61450f954cb9fa317f2/spec/views/devise/sessions/new.html.erb_spec.rb#L217-L218

```suggestion
```